packages/@expressive-code/plugin-shiki/test/rendering.test.ts

```diff
@@ -66,4 +80,25 @@ describe('Renders syntax highlighting', () => {
 		},
 		{ timeout: 5 * 1000 }
 	)
+
+	test(
+		'Supports ansi',
+		async ({ meta: { name: testName } }) => {
+			await renderAndOutputHtmlSnapshot({
+				testName,
+				testBaseDir: __dirname,
+				fixtures: buildThemeFixtures(themes, {
+					code: ansiTestCode,
+					language: 'ansi',
+					meta: '',
+					plugins: [pluginShiki()],
+					blockValidationFn: ({ renderedGroupAst }) => {
+						const html = toHtml(renderedGroupAst)
+						expect(html).not.toMatch(ansiEscapeCode)
```

Comment:
See comment above. In addition to the output not containing any ANSI escape codes anymore, we should also validate at least one resulting ANSI color being present in the HTML output of one theme.